test/controllers/orders_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class OrdersControllerTest < ActionDispatch::IntegrationTest
```

Comment:
I think you can remove those. We use rspec + factory bot for our testing suite if you want to write tests :))